tools/ete/ete_mod.xml

```diff
@@ -0,0 +1,170 @@
+            truevalue="--sort_branches" falsevalue=""
+            help="Sort branches according to node names (--sort_branches)" />
+        <param name="ladderize" type="boolean" label="Ladderize"
+            truevalue="--ladderize" falsevalue=""
+            help="Sort branches by partition size (--ladderize)" />
+        <param name="resolve_polytomies" type="boolean" label="Resolve polytomies"
+            truevalue="--resolve_polytomies" falsevalue=""
+            help="Converts polytomies into bifurcations (--resolve_polytomies)" />
+        <param name="standardize" type="boolean" label="Standardize tree"
+            truevalue="--standardize" falsevalue=""
+            help="Standardize tree topology by expanding polytomies and single child nodes (--standardize)" />
+    </inputs>
+    <outputs>
+        <data name="output_tree" format="nhx" label="${tool.name} on ${on_string}"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree1.nhx" />
+            <param name="outgroup" value="A" />
+            <output name="output_tree" file="tree1_ogA.nhx" />
+        </test>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree1.nhx" />
+            <param name="prune" value="A C" />
+            <output name="output_tree" file="tree1_pruneAC.nhx" />
+        </test>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree1.nhx" />
+            <param name="prune" value="A C" />
+            <param name="prune_preserve_lengths" value="--prune_preserve_lengths" />
+            <output name="output_tree" file="tree1_pruneACpreserve.nhx" />
+        </test>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree1.nhx" />
+            <param name="unroot" value="--unroot" />
+            <output name="output_tree" file="tree1_unroot.nhx" />
+        </test>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree2.nhx" />
+            <param name="sort_branches" value="--sort_branches" />
+            <output name="output_tree" file="tree2_sort.nhx" />
+        </test>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree1.nhx" />
+            <param name="ladderize" value="--ladderize" />
+            <output name="output_tree" file="tree1_ladder.nhx" />
+        </test>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree1_unroot.nhx" />
+            <param name="resolve_polytomies" value="--resolve_polytomies" />
+            <output name="output_tree" file="tree1_unroot_resolve.nhx" />
+        </test>
+        <test>
+            <param name="input_tree" ftype="nhx" value="tree3.nhx" />
+            <param name="standardize" value="--standardize" />
+            <output name="output_tree" file="tree3_stand.nhx" />
+        </test>
+    </tests>
+    <help><![CDATA[
+usage: ete3 mod [-h] [-t [SRC_TREES [SRC_TREES ...]]]
```

Comment:
@bernt-matthias I don't think it's useful to have the tool help copied in the `<help>` section, it's better to have a general description of what the tool does and a link to the upstream documentation. Each option should be documented using the `help` attribute of the `<param>`s.